is_prime.py

```diff
@@ -0,0 +1,17 @@
+import math
+def is_prime(max_number):
+# prime factors
+    i = 0
+    if max_number == 4 or max_number == 1:
+        return False
+
+    for divider in range(2, math.floor((max_number)/2)):
```

Comment:
Maybe use the same trick with /2 as in problem 10?

Reply:
I changed it to sqrt. It is a more accurate approximation.